content/Rustup-1.28.2.md

```diff
@@ -0,0 +1,85 @@
++++
+path = "2025/04/25/Rustup-1.28.2"
+title = "Announcing rustup 1.28.2"
+authors = ["The Rustup Team"]
+aliases = ["2025/04/25/Rustup-1.28.2.html"]
++++
+
+The rustup team is happy to announce the release of rustup version 1.28.2.
+[Rustup][install] is the recommended tool to install [Rust][rust], a programming language that is
+empowering everyone to build reliable and efficient software.
+
+## What's new in rustup 1.28.2
+
+The headlines of this release are:
+
+- The cURL download backend is now officially deprecated and a warning will start to show up
+  when it is used. [pr#4277]
+
+  - While rustup predates reqwest and rustls, the rustup team has long wanted to standardize on
+    an HTTP + TLS stack in Rust, which should increase security, potentially improve performance, and
+    simplify maintenance of the project.
+    With the default download backend already switched to reqwest since [2019][pr#1660], the team
+    thinks it is time to start removing the cURL backend and focus on maintaining the Rust-based
+    stack.
+
+  - The rustup team encourages everyone to switch to the reqwest backend, and would love to hear from
+    you about your use case via [GitHub Issues][issue tracker] if it does work against your particular
```

Comment:
Does *not* work, perhaps?

Reply:
@shepmaster The negation is in "against", but I guess it's a bit too easy to misread? Fixed anyway :)

Reply:
Yes, I would definitely have never read "against" as a negation (and I still don't, even with your response). For example, "I tested my changes against the main branch" has no such negative meaning to me.